modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1547,17 +1556,31 @@ impl Wallet {
         cfg: &WalletConfig,
         task_group: &TaskGroup,
         bitcoind: &DynBitcoindRpc,
-    ) -> (watch::Receiver<Option<u32>>, watch::Receiver<Feerate>) {
+    ) -> anyhow::Result<(watch::Receiver<Option<u32>>, watch::Receiver<Feerate>)> {
         let (block_count_tx, block_count_rx) = watch::channel(None);
         let (fee_rate_tx, fee_rate_rx) = watch::channel(cfg.consensus.default_fee);
 
+        let sources = std::env::var(FM_WALLET_FEERATE_SOURCES_ENV)
+            .unwrap_or_else(|_| match cfg.consensus.network.0 {
+                Network::Bitcoin => "https://mempool.space/api/v1/fees/recommended#.;https://blockstream.info/api/fee-estimates#.%221%22".to_owned(),
```

Comment:
For mainnet, these would be the defaults.

Reply:
~I think I found the problem: you don't use `SafeUrl` for parsing out the fragment, so you don't URL-decode `%221` etc., so jaq can't interpret it. If that's the case it would also point towards not properly handling errors?~

Never mind, second commit changes this